gcp-deployer/CONTRIBUTING.md

```diff
@@ -139,12 +135,13 @@ ERROR: (gcloud.config.get-value) Section [core] has no property [project].
 	```bash
 	$ ./gcp-deployer create -c cluster.yaml -m machines.yaml -s machine_setup_configs.yaml
 	```
-[Optional]To verify API server has been deployed successfully, you can the following command to double check.
     
-    ```bash
+    [Optional]To verify API server has been deployed successfully, you can the following command to double check.
+
+	```bash
```

Comment:
Minor nit: This isn't bash, this is a console session. It should use `console` after the triple backtick instead of bash. Here's the difference:

*console*
```console
foo@bar$ kubectl get pods
here are some pods
```

*bash*
```bash
foo@bar$ kubectl get pods
here are some pods
```